pkg/upload/js.go

```diff
@@ -27,12 +189,37 @@ func ProcessJs(
 	logger log.Logger,
 ) error {
 
-	var rootDirPath string
+	for _, path := range Path {
+
+		outputPath := path
+
+		// Dist is the default output path for webpack
+		if utils.IsDir(filepath.Join(path, "dist")) {
+			outputPath = filepath.Join(path, "dist")
+		}
+
+		// Set a default value for projectRoot if it's not defined
+		projectRoot := resolveProjectRoot(projectRoot, path)
+
+		appVersion, err := ResolveVersion(versionName, projectRoot, logger)
```

Comment:
We should keep the concept of "project root" as being _only_ the path passed to the API. For everything else we use the `path`.

The distinction is that the latter is expected to be on the filesystem of the machine running the command; whereas the latter could be separate machine that built the source map and isn't accessible during the upload.